torch/nn/modules/flatten.py

```diff
@@ -53,16 +53,16 @@ class Unflatten(Module):
       be either `int` or `str` when `Tensor` or `NamedTensor` is used, respectively.
 
     * :attr:`unflattened_size` is the new shape of the unflattened dimension of the tensor and it can be
-      a `tuple` of ints or `torch.Size` for `Tensor` input or a `NamedShape` (tuple of `(name, size)` tuples)
-      for `NamedTensor` input.
+      a `tuple` of ints or `torch.Size` for `Tensor` input or the size as a list or a `NamedShape` 
```

Comment:
This update should explaing that unflattened size can now take a tuple or list of ints or a torch.Size for Tensor input. I wouldn't fix the NamedTensor case in this PR. You'd have to add more tests for it, anyway.